packages/builder/src/components/design/settings/controls/ButtonActionEditor/actions/ScrollTo.svelte

```diff
@@ -36,6 +36,15 @@
   />
   <Label small>Field</Label>
   <Combobox bind:value={parameters.field} options={fieldOptions} />
+  <Label small>Vertical alignment</Label>
+  <Select
+    bind:value={parameters.block}
+    options={[
+      { label: "Start", value: "start" },
+      { label: "Center", value: "center" },
+      { label: "End", value: "end" },
+    ]}
+  />
```

Comment:
I'd prefer to avoid adding this option if possible - I'm a strong believe that this should always be `nearest` which will scroll as little as possible to show your component. It means it accounts for needing to scroll either up or down to see the field. Picking any of these options will always overscroll for at least one direction. So I think we don't need to provide this option here, and instead we just always use `block: "nearest"` in the button action handler.